core/src/main/java/edu/wpi/grip/core/Main.java

```diff
@@ -23,22 +21,13 @@
  */
 public class Main {
 
-    @Inject
-    private Project project;
-    @Inject
-    private PipelineRunner pipelineRunner;
-    @Inject
-    private EventBus eventBus;
-    @Inject
-    private Logger logger;
+    @Inject private Project project;
+    @Inject private PipelineRunner pipelineRunner;
+    @Inject private EventBus eventBus;
+    @Inject private Logger logger;
 
     @SuppressWarnings("PMD.SystemPrintln")
     public static void main(String[] args) throws IOException, InterruptedException {
-        // Close GRIP when we get SIGHUP.  This signal is sent, for example, when GRIP is run in an SSH session
-        // and the session is closed.
-        Signal.handle(new Signal("HUP"), signal -> SafeShutdown.exit(0));
```

Comment:
Now GRIP on a remote device will continue to publish after you disconnect.
The deployed script should respond to the hangup instead.